### ISSUE TITLE
Missed grandslam

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#225]
80DEDF14BD4E83A41980DE2B80C7
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/4fea0398-c059-484a-a415-246cfce9cf41)

So many things I do not like here.

1H-2D is very fine
Now 3C is showing extra, so it should either bid 2H or 3D (I prefer 3D)

Now pick a slam is just giving up on bidding the grand

Now if I bid 3D, the bidding will be
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/ebebede4-670f-455b-8fa5-af2b256ba49f)

And suddenly found 7N that can't make.

Same happends after bidding 2H, and now I think we have another problem, as 3-card support must be shown, and BEN is trained to bid 3C, that is a very wrong bid

This is how Q-Plus bid it:
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/82f2743c-1847-4f3e-8a0a-2ff33ae18175)

And this is how BEN missed it

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/976bdd8c-d2cc-4144-8e76-ab9c2226a4ff)

**Comment by EdwardPiwowar** [comment]
Yes, could be much better but I don't know how to avoid 7NT, only solely based on IMP scoring?
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/6910fa14-7f6a-4598-b2a6-08cf74ea1c7d)

**Comment by ThorvaldAagaard** [comment]
That is a problem in your single dummy analysis. If you used percentages for making the different contracts, it would probably be 7N 60-70% and 7D 80-90%. In matchpoint we bid 7N, but at Imp 7D is much better - this is all about the risk of going down

**Comment by EdwardPiwowar** [comment]
Using SD without DD support is very difficult to estimate percentages.
By the way, the IMP score option should be used to train BEN, I'm not sure if this is the case.

**Comment by ThorvaldAagaard** [comment]
Where do you set the IMP score option?

**Comment by EdwardPiwowar** [comment]
The player's context menu:
https://sites.google.com/view/bbaenglish/program-description/other-program-options

**Comment by ThorvaldAagaard** [comment]
I have never changed it. But will for the future - was MP the default?

**Comment by EdwardPiwowar** [comment]
MP is default but your selection is stored.

**Comment by ThorvaldAagaard** [comment]
I better stop my training then :-)

Will you make a new release later today?

**Comment by EdwardPiwowar** [comment]
I can upload it, but I should also check if there is any regression.

**Comment by ThorvaldAagaard** [comment]
I will just check before I go to bed, then it can generate during night

**Comment by EdwardPiwowar** [comment]
ok

**Comment by ThePokerDude** [comment]
The bidding got worse.
7nt is a special contract. You should know enough to be able to calculate safe 12 tricks at least.
What do we know here:
Partner has Ah and Qd. That makes 9 tricks.
In 7nt we need 4 more sure tricks. I think that's against the odds.
For 7d on the other hand we have far more options. Spade ruffs, possibility to ruff high the hearts.
![image](https://github.com/user-attachments/assets/03f7892f-00db-47d1-94b5-73dc0f04758e)
